blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -111,54 +41,45 @@ export function PresentationalNav(props) {
 		primaryLogoAlt,
 	} = props;
 	return (
-		<StyledNav
+		<Stack
 			id="main-nav"
-			className={navColorClass}
-			navBarBackground={backgroundColor}
-			navHeight={navHeight}
-			scrolled={scrolled}
-			// hard-coded to medium-breakpoint
-			breakpoint={mediumBreakpoint}
+			className={BLOCK_CLASS_NAME}
 			aria-label={sectionAriaLabel}
+			direction="horizontal"
+			alignment="center"
 		>
-			<div
-				className={`news-theme-navigation-container news-theme-navigation-bar logo-${logoAlignment} ${
-					displayLinks ? "horizontal-links" : ""
-				}`}
+			<NavSection
+				customFields={customFields}
+				menuButtonClickAction={menuButtonClickAction}
+				side="left"
+				signInOrder={signInOrder}
 			>
-				<NavSection
-					customFields={customFields}
-					menuButtonClickAction={menuButtonClickAction}
-					side="left"
-					signInOrder={signInOrder}
-				>
-					{children}
-				</NavSection>
-				<NavLogo
-					alignment={logoAlignment}
-					imageSource={primaryLogoPath}
-					imageAltText={primaryLogoAlt}
-					mediumBreakpoint={mediumBreakpoint}
+				{children}
+			</NavSection>
+			<NavLogo
+				alignment={logoAlignment}
+				imageSource={primaryLogoPath}
+				imageAltText={primaryLogoAlt}
+				mediumBreakpoint={mediumBreakpoint}
+			/>
+			{displayLinks ? (
```

Comment:
if there's a manual opt-out of showing links, then this would need to be a bem modifier, for instance `b-horizontal-links-bar--hidden`.

Reply:
@JackHowa This was fixed and now its broke again.  I'll have to look at it on friday.

Reply:
Medium breakpoint has been removed

Reply:
> Medium breakpoint has been removed

it has been removed? I still see mediumBreakpoint. Maybe I've lost context for this comment. But it looks like it's using the default and never actually being set 

<img width="828" alt="Screen Shot 2022-07-15 at 10 16 01" src="https://user-images.githubusercontent.com/5950956/179193727-3c5a3c78-45c5-4032-a872-c58cd8dcca3a.png">